src/test/kotlin/io/github/kryszak/gwatlin/util/ResourcesUtil.kt

```diff
@@ -0,0 +1,13 @@
+package io.github.kryszak.gwatlin.util
+
+object ResourcesUtil {
```

Comment:
Let's make this object internal just for codacy complains

Reply:
Fixed :+1: